src/devices/Mcp3008/Mcp3008.cs

```diff
@@ -29,41 +23,36 @@ public void Dispose()
             _spiDevice = null;
         }
 
-        public int Read(int channel, InputConfiguration inputConfiguration = InputConfiguration.SingleEnded)
+        public int Read(int channel)
```

Comment:
Given that you are already here and that we want to support this binding officially, can you add xml docs to all of the three public methods and to the class?